Automattic/simplenote-ios
# Dark Mode: Mark VI

### Details:
- This PR brings Dark Mode support to the Share Extension. It'll always match the system's UsertInterface Style
- Simplenote's Colors have been moved to their own Assets Catalog: `Colors.xcassets`
- Few files have been shared between the Main App <> Share Extension (`UIColorName`, `VSTheme+Keys`, and `Colors.xcassets`)

Closes #367
**YES** this should be it!!

cc @bummytime (Thank you!!!)


### Testing: iOS 13
1. Launch simplenote on an iOS 13 device
2. Log into your account
3. Switch over to Safari
4. Trigger the Share Extension

- [x] Verify it matches the System's Style
- [x] Switch the Simulator to Dark Mode: **Settings** > **Developer** > **Dark Mode**
- [x] Repeat steps 3-5 and verify the Share Extension now is Dark

### Testing: iOS 12
1. Launch simplenote on an iOS 12 device
2. Log into your account
3. Switch over to Safari
4. Trigger the Share Extension

- [x] Verify everything looks as expected

### Testing: iOS 10
1. Launch simplenote on an iOS 10 device
2. Log into your account
3. Switch over to Safari
4. Trigger the Share Extension

- [x] Verify everything looks as expected

### Testing: Regressions
- [x] Please run a smoke test on iOS 13 in both, Light and Dark mode
- [x] Run the same smoke test on iOS 12

No code from the main app was changed. **However** we did relocate all of the assets, better to be safe than sorry!
